spack-environments/isambard-a64fx/compute-node/spack.yaml

```diff
@@ -8,6 +8,9 @@ spack:
   view: true
   include:
   - ../../common.yaml
+  concretizer:
+    targets:
+      host_compatible: False
```

Comment:
For the record, this is necessary only because Spack misdetects the A64FX CPU on Isambard as generic Aarch64, because it uses an old Cray programming environment, see

* https://github.com/archspec/archspec-json/pull/59#issuecomment-1339279316
* https://github.com/archspec/archspec-json/issues/23#issuecomment-1340882940
* https://github.com/archspec/archspec-json/issues/23#issuecomment-1415799814